src/mmds/src/data_store.rs

```diff
@@ -15,16 +15,19 @@ pub struct Mmds {
 #[derive(Debug, PartialEq)]
 pub enum Error {
     NotFound,
+    NotInitialized,
     UnsupportedValueType,
 }
 
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match *self {
             Error::NotFound => write!(f, "The MMDS resource does not exist."),
-            Error::UnsupportedValueType => {
-                write!(f, "Cannot add non-strings values to the MMDS data-store.")
-            }
+            Error::NotInitialized => write!(f, "The MMDS data store is not initialized."),
+            Error::UnsupportedValueType => write!(
+                f,
+                "Retrieval of the requested resource value type is not supported."
```

Comment:
nit: "Cannot retrieve value. The value has an unsupported type."

Reply:
Done.